src/extractors/jsxtag-gettext.js

```diff
@@ -1,10 +1,57 @@
 import * as t from 'babel-types';
 import gettext from './tag-gettext';
+import { PO_PRIMITIVES } from '../defaults';
+import { template2Msgid, msgid2Orig, hasExpressions,
+    ast2Str, getQuasiStr, strToQuasi, dedentStr } from '../utils';
+import { NoTranslationError } from '../errors';
 
+const { MSGSTR } = PO_PRIMITIVES;
 const NAME = 'jsxtag-gettext';
 
 function match(node, context) {
     return t.isTaggedTemplateExpression(node) && node.tag.name === context.getAliasFor(NAME);
 }
 
-export default { ...gettext, match, name: NAME };
+function resolveDefault(nodePath, poData, context) {
```

Comment:
resolveDefault is invoked in 2 cases:
1. if no translation was found
2. if c-3po is nor in resolving, nor in extracting mode. (Just strips c-3po functions).